src/dashboard.js

```diff
@@ -1184,6 +1291,7 @@ class Dashboard extends Component {
           </Group>
         </Layer>
         {this.state.loading && loading}
+        {tooltipLayer}

```

Comment:
Can use condition here {this.state.showTooltip && tooltipLayer} So if showTooltip is false, the component will not load and render to save cost

Reply:
good suggestion